frontend/scss/components/templates/what-is-amp.scss

```diff
@@ -36,6 +36,10 @@
 
   .#{utility('container-fluid')} {
     @include container-fluid;
+    
+    > * {
```

Comment:
is there any way to make this more specific? This selector would cause a check for every element on every page paint/invalidation

Reply:
or is what-is-amp only built for the one single page (and not included elsewhere)

Reply:
As far as I can see it 'what-is-amp' is only used for the four /about-pages. So this shouldn't cause a problem anywhere else on the site.

Reply:
Hm... `> div, section` might probably still be enough? 😁  Comment out

https://github.com/ampproject/amp.dev/blob/68e80bcacb70eb2c7d104552f8b1b820d2377d32/platform/lib/utils/ampOptimizerConfig.js#L28

for easier debugging.

Reply:
Alright, alright. Valid point. 😄  Please check again.